Dhali-org/Dhali-document-qa
# Determine cost of usage

## What does this PR do

* Adds Firebase document retrieval functionality to determine cost of asset usage in real-time

## How should this PR be tested?

* Spin up the application
* Ensure costs are presented correctly

## Checklist before requesting a review
